src/handlers/localstorage/common.ts

```diff
@@ -46,14 +116,13 @@ export const getLocal = async (key = '', version = defaultVersion) => {
     });
     if (result && result.storageVersion === version) {
       return result;
-    }
-    if (result) {
-      removeLocal(key);
+    } else if (result && result.storageVersion !== version) {
```

Comment:
wouldn't need the additional `&& result.storageVersion !== version` check here given the result would've returned above on line 118